### PR TITLE
ci(env config, template):

### PR DESCRIPTION
**Pull request by mihai-ro**
*  Remove separate env files.
* Update dependencies.
* Update configureEnv func.
* Update pseudomodule style to fit latest kogaio.
* Create web-component template.

### CODE REVIEW

File: templates/web/.env-cmdrc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "REACT_APP_USER_POOL_ID": "USER_POOL_ID_PLACEHOLDER",
+    "REACT_APP_USER_POOL_WEB_CLIENT_ID": "USER_POOL_WEB_CLIENT_ID_PLACEHOLDER",
+    "REACT_APP_AWS_APPSYNC_GRAPHQL_ENDPOINT": "APPSYNC_GRAPHQL_ENDPOINT_PLACEHOLDER",
+    "REACT_APP_AWS_APPSYNC_REGION": "us-east-1",
+    "REACT_APP_AWS_APPSYNC_AUTHENTICATION_TYPE": "AMAZON_COGNITO_USER_POOLS"
+  },
+  "staging": {
+    "REACT_APP_REGION": "us-east-1",
+    "REACT_APP_USER_POOL_ID": "1_AolIaFtPK",
+    "REACT_APP_USER_POOL_WEB_CLIENT_ID": "hec5fdcecamsh20ipr3lv4401",
```
Comment (BvmvB) [inline templates/web/.env-cmdrc]: Hardcoded data that I think should not be there.

File: templates/web-component/apps/PLACEHOLDER.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  <Flex alignItems='center' justifyContent='center'>
+    <Typography variant='h3' {...props}>
+      Hello from PLACEHOLDER
+    </Typography>
+  </Flex>
+)
+
+PLACEHOLDER.propTypes = {}
+
+export default PLACEHOLDER
```
Comment (BvmvB) [inline templates/web-component/apps/PLACEHOLDER.jsx]: What is the use case for this component? 🤔

File: templates/web-component/package.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "PLACEHOLDER",
```
Comment (BvmvB) [inline templates/web-component/package.json]: I realized that we didn't discuss this. The CLI will replace PLACEHOLDER with the project name and the key is PROJECT_NAME.

File: templates/web/.env-cmdrc (hunk 1 above)
Comment (mihai-ro) [reply templates/web/.env-cmdrc]: yes, you are right. my bad:)

File: templates/web-component/apps/PLACEHOLDER.jsx (hunk 2 above)
Comment (mihai-ro) [reply templates/web-component/apps/PLACEHOLDER.jsx]: an entry component, just so the developer would expect something to be rendered when importing the web component

File: templates/web-component/package.json (hunk 3 above)
Comment (mihai-ro) [reply templates/web-component/package.json]: Oh, got it. thanks; will make the changes in a jiffy

**Pull request merged**